File: results/java/MINIMUM_COST_MAKE_ARRAY_SIZE_1_REMOVING_LARGER_PAIRS.cpp
int f_filled ( int a [ ] , int n ) {
  int min = a [ 0 ] ;
  for ( int i = 1 ;
  i < a [ 0 ] ;
  i ++ ) {
    if ( a [ i ] < min ) {
      min = a [ i ] ;
    }
  }
  return ( n - 1 ) * min ;
}

File: TransCoder_tokenized_test_set_functions/java/MINIMUM_COST_MAKE_ARRAY_SIZE_1_REMOVING_LARGER_PAIRS.java
static int cost ( int [ ] a, int n ) {
  int min = a [ 0 ];
  for ( int i = 1;
  i < a . length;
  i ++ ) {
    if ( a [ i ] < min ) min = a [ i ];
  }
  return ( n - 1 ) * min;
}